segmentstore/server/src/main/java/io/pravega/segmentstore/server/tables/ContainerKeyIndex.java

```diff
@@ -697,6 +745,43 @@ void updateSegmentIndexOffset(long segmentId, long indexOffset, boolean cacheUpd
                     task.task.complete(cacheUpdated);
                 }
             }
+
+            if (throttler != null) {
+                if (removed) {
+                    // We evicted the segment, so close the throttler.
+                    throttler.close();
+                } else {
+                    // We have made indexing progress. Update the throttler with the count.
+                    throttler.release(Math.max(0, processedEntryCount));
```

Comment:
I'm not being able to follow properly the call path that leads to this line here. I checked it up to here:

https://github.com/pravega/pravega/pull/3792/files#diff-8e73c522398ad0c82ca209a786c9c920R269

and I don't see how the credits that we are releasing here have been added to the throttler. Where is the associated execution?